src/stores/MorseStore.js

```diff
@@ -14,6 +14,7 @@ class MorseStore extends SettingsSaver {
       effectiveSpeed: 30,
       characterSpeed: 30,
       frequency: 500,
+      automaticallyRepeat: true,
```

Comment:
FYI: I missed this in all the PRs so far, but these settings should live in ReadTrainerStore instead. I fixed them all up now.

Reply:
Ah. Good catch. I didn't even realize that either.